src/ui/screens/home/servers/ServerCountry.qml

```diff
@@ -22,7 +22,7 @@ MZClickableRow {
     property var currentCityIndex
     property alias serverCountryName: countryName.text
     property var cityList: cityListVisible ? cityLoader.item : cityLoader
-    property var busy: cityListVisible ? scrollAnimation.running : false
+    property var busy: cityListVisible && (scrollAnimation.running || listOpenTransition.running || listClosedTransition.running)
```

Comment:
nit: Does something like this work on QML?

```suggestion
    property var busy: cityListVisible && transitions.find(t => t.running)
```

Reply:
This construction does not seem to work in QML. It generates an error:
`qrc:/ui/screens/home/servers/ServerCountry.qml:25: TypeError: Property 'find' of object [object Object] is not a function (ServerCountry.qml:25)`

Reply:
Looks like translations is not an array although it looks like one. 🤷‍♀️